src/options.cpp

```diff
@@ -761,13 +761,6 @@ void initOptions()
                                  );
 
     ////////////////////////////GRAPHICS/////////////////////////
-    OPTIONS["NO_BRIGHT_BACKGROUNDS"] = cOpt("graphics", _("No bright backgrounds"),
-                                            _("If true, bright backgrounds are not used - some consoles are not compatible."),
-                                            false
```

Comment:
The incompatibility is that some terminal emulators understand the bright
background flag as bold, to the best of my knowledge. That won't prevent
anyone from accessing the options menu.

On Thu, Apr 2, 2015 at 5:06 AM KA101 <notifications@github.com> wrote:

> In src/options.cpp
> <https://github.com/CleverRaven/Cataclysm-DDA/pull/11878#discussion_r27628344>
> :
>
> > @@ -761,13 +761,6 @@ void initOptions()
> >                                   );
> >
> >      ////////////////////////////GRAPHICS/////////////////////////
> > -    OPTIONS["NO_BRIGHT_BACKGROUNDS"] = cOpt("graphics", _("No bright backgrounds"),
> > -                                            _("If true, bright backgrounds are not used - some consoles are not compatible."),
> > -                                            false
>
> Hmm. Just for safety's sake, how will this affect
> bright-background-incompatible consoles--will they be able to access the
> menu and enable the template (and then later modify it, in the event that
> the user happens to need/want modified colors)?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CleverRaven/Cataclysm-DDA/pull/11878/files#r27628344>.
>